js/values/AssessmentResult.js

```diff
@@ -43,4 +55,13 @@ AssessmentResult.prototype.getText = function() {
 	return this.text;
 };
 
+/**
+ * Set the text for the assessment.
+ * @param {string} text The text to be used for the text property
+ * @returns {void}
+ */
+AssessmentResult.prototype.setText = function( text ) {
+	this.text = text;
```

Comment:
We could include a check for an undefined string, and set it as an empty string, so it matches the default.